src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigCommandExecutor.java

```diff
@@ -0,0 +1,44 @@
+package io.sealights.plugins.sealightsjenkins.buildsteps.cli.executors;
+
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.entities.BaseCommandArguments;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.entities.PrConfigCommandArguments;
+import io.sealights.plugins.sealightsjenkins.utils.Logger;
+
+import java.util.List;
+
+public class PrConfigCommandExecutor extends ConfigCommandExecutor{
+    private PrConfigCommandArguments prConfigCommandArguments;
+    public PrConfigCommandExecutor(Logger logger, BaseCommandArguments baseCommandArguments,
+     PrConfigCommandArguments prConfigCommandArguments) {
+        super(logger, baseCommandArguments, prConfigCommandArguments);
+        this.prConfigCommandArguments = prConfigCommandArguments;
+    }
+
+    @Override
+    protected String getCommandName() {
+        return "-prConfig";
+    }
+
+    @Override
+    protected void addBaseArgumentsLine(List<String> commandsList) {
+        if (baseArgs.getTokenData() != null) {
+            addArgumentKeyVal("token", baseArgs.getTokenData().getToken(), commandsList);
+        } else {
+            addArgumentKeyVal("tokenfile", baseArgs.getTokenFile(), commandsList);

```

Comment:
Are you sure you'll always have token file in this case?

Reply:
Yes, token data resolved in the beginning of ```CLIRunner```